### PR TITLE
[ewasm] add ewasm bytecode compare support.

### PR DESCRIPTION
**Pull request by aarlt**
- just a test for now
- todo: emscripten version still don't generate ewasm code

### CODE REVIEW

File: scripts/bytecodecompare/prepare_report.py [hunk 1]
```diff
@@ -10,34 +10,41 @@
...
-        input_json = {
-            'language': 'Solidity',
-            'sources': sources,
-            'settings': {
-                'optimizer': {
-                    'enabled': optimize
-                },
-                'outputSelection': {'*': {'*': ['evm.bytecode.object', 'metadata']}},
-                'modelChecker': { "engine": 'none' }
+        for output_type in [['evm.bytecode.object', 'metadata'], ['ewasm.wast', 'ewasm.wasm', 'metadata']]:
```
Comment (aarlt) [inline scripts/bytecodecompare/prepare_report.py:13]: looks like that `ewasm.wasm` only works, if `ewasm.wast` is set

**Comment by cameel** [comment]
As discussed on gitter, I'm taking over this PR and I'll integrate it with my other bytecode compare PRs.

**Comment by chriseth** [comment]
I would say let's postpone this for now. Bytecode generate for ewasm takes too long to run for all files.

**Comment by chriseth** [comment]
Closing this for now.

**Pull request closed without merging**